db/migrate/20180720095520_add_status_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :status, :integer, default: 1
```

Comment:
why do we have default 1 here?

Reply:
This is because when a new user will sign in, he will be enable by default. 
As you have told me to add new user as disable by default, I have changed the migration.